README.md

```diff
@@ -1,102 +1,111 @@
-![Logo](https://github.com/datatrails/datatrails-scitt-samples/blob/main/DataTrails_Horizontal_Logo_Black.png#gh-light-mode-only)
-![Logo](https://github.com/datatrails/datatrails-scitt-samples/blob/main/DataTrails_Horizontal_Logo_White.png#gh-dark-mode-only)
+![Logo](https://raw.githubusercontent.com/datatrails/datatrails-scitt-samples/main/DataTrails_Horizontal_Logo_Black.png)
+![Logo](https://raw.githubusercontent.com/datatrails/datatrails-scitt-samples/main/DataTrails_Horizontal_Logo_White.png)
 
-# datatrails-scitt-samples
+# DataTrails SCITT Examples
 
-The files in this repository can be used to demonstrate how the DataTrails SCITT API works. To do this please download one or more of these files, then follow
+The files in this repository can be used to demonstrate how the DataTrails SCITT API works.
 
-the getting started docs.
+## Copy the Samples
 
-## Generating a signing key
+```shell
+git clone https://github.com/datatrails/datatrails-scitt-samples.git
+
+cd datatrails-scitt-samples
+```
+
+## Generating a Signing Key
 
 In the samples we assume the signed key is an ecdsa p256 key:
 
-```
+```shell
 openssl ecparam -name prime256v1 -genkey -out scitt-signing-key.pem
 ```
 
-## Generating a statement
+## Generating a Statement
 
 In the samples we assume the statement is a json document, e.g:
 
-```
+```shell
+cat > statement.json <<EOF
 {
     "author": "fred",
     "title": "my biography",
     "reviews": "mixed"
 }
+EOF
 ```
 
-## Creating a signed statement
+## Creating a Signed Statement
 
 To create a signed statement we can use a venv.
 
 Create a new venv:
 
-```
-python -m  venv
+```shell
+python -m  venv venv
 ```
 
 Now activate the new venv:
 
-```
+```shell
 source venv/bin/activate
 ```
 
 Now ensure all the requirements are installed:
 
-```
+```shell
+pip install --upgrade pip && \
 pip install -r requirements.txt
 ```
 
 Finally we have an environment we can run the create signed statement script in:
 
-```
-python scitt/create_signed_statement.py --signing-key-file scitt-signing-key.pem \
-    --statement-file scitt-statement.json \
-    --feed testfeed \
-    --issuer testissuer \
-    --output-file signed-statement.txt
+```shell
+python scitt/create_signed_statement.py \
+    --signing-key-file scitt-signing-key.pem \
+    --payload statement.json \
+    --feed my-product-name \
+    --issuer sysnation.dev \
```

Comment:
changing the issuer back to an example issuer for the sample repo, as this is a general purpose script.

I think we agreed we would just scrap this who readme in favour of just liking to the quickstart scitt docs?

Reply:
yes, we'll update the readme to point to: https://docs.datatrails.ai/developers/developer-patterns/scitt-api/